app/assets/stylesheets/govuk_publishing_components/components/_lead-paragraph.scss

```diff
@@ -12,3 +12,10 @@
 .gem-c-lead-paragraph--inverse {
   color: govuk-colour("white");
 }
+
+@include govuk-media-query($media-type: print) {
+  .gem-c-lead-paragraph {
+    color: $govuk-print-text-colour;
+    background-color: unset;
```

Comment:
I don't think we need to set a background colour here - inverse components don't set a background colour, they sit on elements that already specify the colour.

Reply:
OK, removed.